feature-libs/cart/base/components/cart-shared/cart-item/cart-item.component.html

```diff
@@ -88,7 +88,7 @@
               [readonly]="
                 !item.updateable || readonly || options.isSaveForLater
               "
-              [max]="item.product.stock?.stockLevel"
+              [max]="item.product?.stock?.stockLevel || 1000"
```

Comment:
Let's review this

Reply:
Removing the default value.  It was removed a while ago on develop, but had not been ported to the cart-lib file.
https://github.com/SAP/spartacus/pull/14086/files